src/solvers/flattening/boolbv_get.cpp

```diff
@@ -143,15 +144,14 @@ exprt boolbvt::bv_get_rec(
       for(const auto &c : components)
       {
         const typet &subtype = c.type();
-        op.push_back(nil_exprt());
 
-        std::size_t sub_width=boolbv_width(subtype);
+        const member_exprt member(expr, c.get_name(), subtype);
+        op.push_back(
```

Comment:
Doesn't this then produce a member of nil?

Reply:
That was me being lazy, which I shouldn't have been. Now passing a proper `index_exprt`.